api/host.go

```diff
@@ -22,10 +22,82 @@ const (
 	UsabilityFilterModeUnusable = "unusable"
 )
 
+var validHostSortBy = map[string]any{
```

Comment:
I like sort options as much as the next guy, but this seems excessive? We can always expose extra sort options, but taking them away would be breaking compatibility. Especially the cost/price fields would be returned in lexicographical order.

Edit: hm I saw that you added a test that sorts on a cost field so I guess you considered this. Either that or the sort ended up being numerical by chance.

Reply:
I updated the PR to not sort anymore. Keeping the refactor that I added because of it but it's probably best to wait for v2 before we do sorting since the sorts will also be more useful then. e.g. sorting by download price will be one-dimensional instead of depend on 10 different fields in the price table.